src/Plugin/Field/FieldType/SkosConceptEntityReferenceItem.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\rdf_skos\Plugin\Field\FieldType;
+
+use Drupal\Core\Field\Plugin\Field\FieldType\EntityReferenceItem;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * A field type used to reference SKOS Concept entities.
+ *
+ * It only supports referencing SKOS Concept entities but can be configured
+ * to limit these by Concept Scheme.
+ *
+ * @FieldType(
+ *   id = "skos_concept_entity_reference",
+ *   label = @Translation("SKOS Concept Reference"),
+ *   description = @Translation("References SKOS Concepts."),
+ *   category = @Translation("SKOS"),
+ *   default_widget = "entity_reference_autocomplete",
+ *   default_formatter = "entity_reference_label",
+ *   list_class = "\Drupal\Core\Field\EntityReferenceFieldItemList",
+ * )
+ */
+class SkosConceptEntityReferenceItem extends EntityReferenceItem {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function defaultStorageSettings(): array {
+    return [
+      'target_type' => 'skos_concept',
+    ] + parent::defaultStorageSettings();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function storageSettingsForm(array &$form, FormStateInterface $form_state, $has_data): array {
+    return [];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getPreconfiguredOptions(): array {
+    // We don't want to use this field with any other entity types so we don't
+    // preconfigure anything here.
+    return [];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function fieldSettingsForm(array $form, FormStateInterface $form_state): array {
+    $field = $form_state->getFormObject()->getEntity();
+
+    /** @var \Drupal\Core\Entity\EntityReferenceSelection\SelectionPluginManagerInterface $selection_manager */
+    $selection_manager = \Drupal::service('plugin.manager.entity_reference_selection');
+
+    $form = [
+      '#type' => 'container',
+      '#process' => [[get_class($this), 'fieldSettingsAjaxProcess']],
```

Comment:
`get_class($this)` is called more than twice, it would be nice to use a variable for this.